compiler/src/dmd/traits.d

```diff
@@ -1081,56 +1102,59 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
                  e.ident == Id.getVirtualMethods ||
                  e.ident == Id.getOverloads)
         {
-            uint errors = global.errors;
-            Expression eorig = ex;
-            ex = ex.expressionSemantic(scx);
-            if (errors < global.errors)
-                e.error("`%s` cannot be resolved", eorig.toChars());
-
-            if (e.ident == Id.getVirtualFunctions)
+            if (e.ident != Id.getOverloads || dim != 1)
             {
-                // @@@DEPRECATED2.121@@@
-                // Deprecated in 2.101 - Can be removed from 2.121
-                e.deprecation("`traits(getVirtualFunctions)` is deprecated. Use `traits(getVirtualMethods)` instead");
-            }
+                uint errors = global.errors;
+                Expression eorig = ex;
+                ex = ex.expressionSemantic(scx);
+                if (errors < global.errors)
+                    e.error("`%s` cannot be resolved", eorig.toChars());
 
-            /* Create tuple of functions of ex
-             */
-            auto exps = new Expressions();
-            Dsymbol f;
```

Comment:
what happened to `f`? It's used later on.

Reply:
It's set to `null` way above and filled in directly with the parameter symbol in the one-argument case.